### PR TITLE
Collapse purple theme colors

### LINKED ISSUE #259: Collapse purple colors and make them primary

### PR DESCRIPTION
**Pull request by fbarl**
UI components part of #259.

After some tiny normalizations, the primary colors spectrum consists of the following colors (rough usage estimates in brackets):

 _hsl(240, 20%, 25%)_ - charcoal (20)
 _hsl(240, 20%, 30%)_ - gunpowder (22)
 _hsl(240, 20%, 35%)_ - mulledWine (4)
 _hsl(240, 20%, 42%)_ - cometDark (1)
 _hsl(240, 20%, 45%)_ - comet (3)
 _hsl(240, 20%, 49%)_ - kimberlyDark (4)
 _hsl(240, 20%, 57%)_ - kimberly (7)
 _hsl(240, 20%, 59%)_ - lavender (44)
 _hsl(240, 20%, 65%)_ - amethystSmoke (14)
 _hsl(240, 20%, 75%)_ - logan (6)
 _hsl(240, 20%, 79%)_ - blueHaze (5)
 _hsl(240, 20%, 90%)_ - mischka (9)
 _hsl(240, 20%, 91%)_ - athens (10)
 _hsl(240, 20%, 95%)_ - athensGray (6)
 _hsl(240, 20%, 98%)_ - whiteSmoke (14)

---

New color system (dropped 4 colors from above):

 _hsl(240, 20%, 25%)_ - purple900
 _hsl(240, 20%, 30%)_ - purple800
 _hsl(240, 20%, 35%)_ - purple700
 _hsl(240, 20%, 45%)_ - purple600
 _hsl(240, 20%, 50%)_ - purple500
 _hsl(240, 20%, 60%)_ - purple400
 _hsl(240, 20%, 65%)_ - purple300
 _hsl(240, 20%, 75%)_ - purple200
 _hsl(240, 20%, 90%)_ - purple100
 _hsl(240, 20%, 95%)_ - purple50
 _hsl(240, 20%, 98%)_ - purple25

image

The naming convention was inspired by [Material Design](https://material.io/design/color/the-color-system.html#) and the discussion in #218.

cc @bia @miklosp 

----

**Note**: I also added the default tooltip info on the colors preview page and relaxed the saturation sorting criterion a bit.

### CODE REVIEW

File: docs/js/Example.js [hunk 1]
```diff
@@ -14,8 +14,8 @@ const Panel = styled.div`
 const PanelHeader = styled.div`
   height: 50px;
   padding-left: 20px;
-  background-color: ${props => props.theme.colors.athens};
-  color: ${props => props.theme.colors.gunpowder};
+  background-color: ${props => props.theme.colors.primary.purple100};
+  color: ${props => props.theme.colors.purple800};
```
Comment (foot) [inline docs/js/Example.js]: `props.theme.colors.primary.purple100` vs `props.theme.colors.purple800`?
+1 on flat! I don't like typing.

Comment (fbarl) [reply docs/js/Example.js]: Yeah, I was also thinking flat could be better, except that it wouldn't be obvious from the code when the theme color is being used.. Not sure about this one still... @miklosp ?

**Comment by miklosp** [comment]
LGTM!

File: docs/js/Example.js (hunk 1 above)
Comment (fbarl) [reply docs/js/Example.js]: I talked with @miklosp and we decided to flatten the main theme colors, starting from this one :)

**Pull request merged**